### PR TITLE
Update 126.net.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/126.net.xml [hunk 1]
```diff
@@ -1,15 +1,19 @@
 <!--
-	Only avaiable in China:	126.net_CN.xml
```
Comment (J0WI) [inline src/chrome/content/rules/126.net.xml]: Why did you remove that file?

File: src/chrome/content/rules/126.net.xml [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 
+	Incomplete cert chain:
+		l.bst.126.net
+-->
 <ruleset name="126.net">
-
 	<target host="img3.126.net" />
 		<test url="http://img3.126.net/kaola/150512/js/jquery-1.4.2.js" />
 	<target host="easyread.ph.126.net" />
-		<test url="http://easyread.ph.126.net/0jkjwGfV7I4EJXnYlRBkVg==/7917077456266817441.jpg" />
```
Comment (J0WI) [inline src/chrome/content/rules/126.net.xml]: Why did you remove this test url? We only have an `AccessDenied` on this host.

File: src/chrome/content/rules/126.net.xml (hunk 1 above)
Comment (ivysrono) [inline src/chrome/content/rules/126.net.xml]: Sorry , I got mismatched 5 days ago in China. My mistake.

**Comment by ivysrono** [comment]
#9471
